src/mode/modeHandler.ts

```diff
@@ -280,10 +295,11 @@ export class SearchState {
     }
   }
 
-  constructor(direction: SearchDirection, startPosition: Position, searchString = "") {
+  constructor(direction: SearchDirection, startPosition: Position, searchString = "", { isRegex = false } = {}) {
```

Comment:
I'm not really sure about this syntax (still a TS noob). Is this what you were thinking?